src/current/cockroachcloud/plan-your-cluster.md

```diff
@@ -8,163 +8,99 @@ cloud: true
-![][image3]
+## Storage pricing
 
-## Examples and Benchmarks for vCPU Capacity
+Storage capacity for a CockroachDB {{ site.data.products.standard }} cluster is [priced]({% link cockroachcloud/billing-management.md %}) on demand in units of GiB-month. Storage prices vary across providers and [regions]({% link cockroachcloud/regions.md %}). Storage is charged based on the logical amount of data in the database.
+- **3 replicas included**: - CockroachDB {{ site.data.products.standard }} maintains at minimum three replicas of your data at no additional cost.
 
-Comparing relative CPU utilization for several benchmarks. Understanding the results below:
+  For a cluster with more than three replicas, storage for each additional replica is priced at the storage price for the region.
 
-### TPC-C
+- **Multi-region storage**: In a multi-region cluster, storage prices vary by region. The data in each region is priced at that region's storage rate.
 
-[TPC](https://www.tpc.org/tpcc/) is an industry standard performance benchmark for OLTP workloads.
+### Single-region example
 
-|  | Standard vCPUs | Advanced vCPUs | vCPU ratio | Standard vCPU cost | Advanced vCPU cost | Standard price-perf savings |
-| :---- | :---- | :---- | :---- | :---- | :---- | :---- |
-| TPC-C (150 warehouses) | 4.98 | 4.86 | 1.03 | $378 | $615 | 39%  |
-| TPC-C (300 warehouses) | 6.51 | 6.51 | 1.00 | $495 | $825 | 40% |
-|  |  |  |  |  |  |  |
+By default, CockroachDB {{ site.data.products.cloud }} maintains three replicas for a single-region cluster, spread across three different availability zones within the region. You are not charged to store these replicas.
 
+For example, if a single-region cluster is deployed in `us-central-1` and contains 500 GiB of data. CockroachDB {{ site.data.products.cloud }} stores three replicas, for a total of 1500 GiB of data. However, you are billed only for 500 GiB of storage.
 
-TODO - remove original table
+If you add a fourth replicas, the cluster consumes 2000 GiB of storage (500 x 4), but you are billed only for 1000 GiB (500 for the data and 500 for the fourth replica).
 
-| TPC-C (150 warehouses) | 4.86 | 4.98 | $61.54 |
-| :---- | ----: | ----: | ----: |
-| TPC-C (300 warehouses) | 6.51 | 6.51 | $60.04 |
-| TPC-C (600 warehouses) | 9.45 | 10.90 | $69.21 |
-| TPC-E (500 customers) | 4.20 | 3.85 | $54.97 |
-| TPC-H (query \#1) | 10.64 | 18.81 | $106.03 |
-| TPC-H (query \#2) | 10.41 | 10.42 | $60.07 |
-| TPC-H (query \#11) | 11.02 | 12.83 | $69.87 |
-| TPC-H (query \#13) | 5.23 | 4.43 | $50.81 |
+### Multi-region example
 
-This table is \[WIP\]
+For multi-region clusters, we recommend 5 or more replicas are typically maintained across availability zones and regions for resilience against both zone and region failure. The first three replicas are included in the base storage price, while additional replicas are priced based on the ratio of additional regions to the initial three.
```

Comment:
```suggestion
For multi-region clusters, we recommend that 5 or more replicas are typically maintained across availability zones and regions for resilience against both zone and region failure. The first three replicas are included in the base storage price, while additional replicas are priced based on the ratio of additional regions to the initial three.
```

also:
* "we typically recommend" or "typically maintained" (i.e. you usually have this, or for most of your tables)?
* starting with the last sentence, I find this section/example hard to follow. May be worth a follow-up post launch to find a clearer way of explaining.

Reply:
Updated your suggested edit, thanks. For your second point, it is confusing but it is explained in the next paragraph. You are charged N/3, so if you have 5 you are charged 2/3 the rate. If you have 6 you are charged 100% the rate. LMK if you think of a better way to put it.